translation/swift/README.md

```diff
@@ -0,0 +1,82 @@
+- [Xcode 10][xcode] or higher
+- [Cocoapods][cocoapods] 
+
+## Setup
+
+- Create a project (or use an existing one) in the [Google Cloud Console][cloud-console]
+- Enable the [Translate API](https://console.cloud.google.com/apis/library/translate.googleapis.com)
+- Enable the [IAM Service Account Credentials API](https://console.cloud.google.com/apis/library/iamcredentials.googleapis.com).
+- [Enable billing][billing]
+- [Create a Service account](https://cloud.google.com/iam/docs/creating-managing-service-accounts) with the following IAM roles: `Cloud Translation API Editor`. Example name: `translation-glossary`. ([For more info on: how to add roles to a Service Account](https://cloud.google.com/iam/docs/granting-roles-to-service-accounts#granting_access_to_a_service_account_for_a_resource))
+- Be sure that you have gone through the steps to use the glossary feature in this sample by [Create glossary](https://cloud.google.com/translate/docs/glossary#create_a_glossary) on your cloud.
+- Go to  [GCP Storage console](https://console.cloud.google.com/storage/) tap on create a new bucket and follow the instructions.
+- Upload the `Example_glossary - Sheet.csv` on your GCP storage's newly created bucket (example .csv file can be find in this `ios-docs-samples/Translation/swift/` directory to test the glossary feature in this sample). 
+
+###  Setup the app
+- Clone this repository `git clone https://github.com/GoogleCloudPlatform/ios-docs-samples.git` 
+- Go to `ios-docs-samples/translation/swift/google/`
+- Copy the below listed folder protos into the `google` folder
+1. api : create `api` folder inside of `ios-docs-samples/translation/swift/google/` folder and paste `annotations.proto`, `http.proto`, `label.proto`, `monitored_resource.proto` from [api](https://github.com/googleapis/googleapis/tree/master/google/api)
+2. Cloud: get the `cloud/translate/v3/translation_service.proto` in your project's google folder. your project structure will look similar to `ios-docs-samples/translation/swift/google/cloud/translate/v3/translation_service.proto` [translation_service.proto](https://github.com/googleapis/googleapis/blob/master/google/cloud/translate/v3/translation_service.proto)
+3. longrunning:  create `longrunning` folder inside of `ios-docs-samples/translation/swift/google/` folder and paste [operations.proto](https://github.com/googleapis/googleapis/blob/master/google/longrunning/operations.proto)
+4. rpc:  create `rpc` folder inside of `ios-docs-samples/translation/swift/google/` folder and paste `code.proto, error_details.proto, status.proto` from [rpc](https://github.com/googleapis/googleapis/tree/master/google/rpc)
+
+- `cd ios-docs-samples/translation/swift/` 
+- Run `./INSTALL-COCOAPODS` to install app dependencies (this can take few minutes to run). When it finishes, it will open the SpeechtoSpeech workspace in Xcode. Since we are using Cocoapods, be sure to open the `Translation.xcworkspace` and not `Translation.xcodeproj`.
```

Comment:
Couldn't this bash script do steps 1-4 for you?

Reply:
No, it will install the Cocoapods and fix the import issues.

Reply:
Yea, but it could also setup the repo and copy over all the files.

Reply:
Done. Added bash script to get the proto files.